jujugui/static/gui/src/app/app.js

```diff
@@ -764,6 +765,7 @@ YUI.add('juju-gui', function(Y) {
           visible={services.length > 0}>
           <components.AddedServicesList
             services={services}
+            getUnitStatusCounts={utils.getUnitStatusCounts.bind(this)}
```

Comment:
the `getUnitStatusCounts` does not reference any instance level code so you can probably avoid the bind here and on 815